[Bug] UT failed at functions_geo

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.


### Version

trunk

### What's Wrong?

1. 
```
In file included from /home/disk3/zxy/baidu/bdg/doris/core/be/src/vec/functions/functions_geo.cpp:18:
/home/disk3/zxy/baidu/bdg/doris/core/be/src/vec/functions/functions_geo.h:43:12: error: 'execute_impl' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
    Status execute_impl(FunctionContext* context, Block& block, const ColumnNumbers& arguments,
           ^
/home/disk3/zxy/baidu/bdg/doris/palo-toolchain/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/type_traits:1372:66: note: in instantiation of template class 'doris::vectorized::GeoFunction<doris::vectorized::StDistanceSphere, doris::vectorized::DataTypeNumber<double>>' requested here
```

2.
```
ons_test.dir/geo_functions_test.cpp.o -c /home/disk3/zxy/baidu/bdg/doris/core/be/test/geo/geo_functions_test.cpp
/home/disk3/zxy/baidu/bdg/doris/core/be/test/geo/geo_functions_test.cpp:332:5: error: use of undeclared identifier 'FLAGS_s2debug'
    FLAGS_s2debug = false;
    ^
1 error generated.
```

### What You Expected?

win

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/be/src/exec/tablet_sink.cpp
+++ b/be/src/exec/tablet_sink.cpp
@@ -551,7 +551,6 @@ void NodeChannel::try_send_batch(RuntimeState* state) {
                                               ReusableClosure<PTabletWriterAddBatchResult>>(
                 &request, _tuple_data_buffer, _add_batch_closure);
     }
-    _add_batch_closure->set_in_flight();
     _stub->tablet_writer_add_batch(&_add_batch_closure->cntl, &request, &_add_batch_closure->result,
                                    _add_batch_closure);
 

--- a/be/src/vec/functions/functions_geo.h
+++ b/be/src/vec/functions/functions_geo.h
@@ -41,7 +41,7 @@ public:
     bool use_default_implementation_for_constants() const override { return true; }
 
     Status execute_impl(FunctionContext* context, Block& block, const ColumnNumbers& arguments,
-                        size_t result, size_t input_rows_count) {
+                        size_t result, size_t input_rows_count) override {
         return Impl::execute(block, arguments, result);
     }
 };

--- a/be/test/geo/geo_functions_test.cpp
+++ b/be/test/geo/geo_functions_test.cpp
@@ -18,6 +18,7 @@
 #include "geo/geo_functions.h"
 
 #include <gtest/gtest.h>
+#include <s2/s2polygon.h>
 
 #include <vector>
 

